modules/denoise/config.py

```diff
@@ -1,5 +1,5 @@
 def can_build(env, platform):
-    return env["tools"]
+    return env["tools"] and "android_arch" not  in env
```

Comment:
Come on, you have `platform` ready to use :)

Reply:
I am too tired for thinking :-P thanks